app/src/users/admin.py

```diff
@@ -116,6 +116,44 @@ def queryset(self, request, queryset):
         return queryset
 
 
+class UserAccommodationListFilter(admin.SimpleListFilter):
+    title = 'user accommodation'
+    parameter_name = 'user_accommodation'
+
+    def lookups(self, request, model_admin):
+        return [
+            ('no', 'No accommodation'),
+            ('not_continuous', "Not continuous accommodation"),
+            ('partial', "Partial accommodation"),
+            ('full', "Full accommodation"),
+        ]
+
+    def queryset(self, request, queryset):
+        value = self.value()
+        if value == 'no':
+            return queryset.filter(
+                Q(preferences__accommodation_day_1=False) &
```

Comment:
I think you don't have to use `Q` objects with `&`, keyword arguments have the same effect, see https://docs.djangoproject.com/en/3.2/ref/models/querysets/#filter

Reply:
So, `Q` object should be used for `or` but not for `and`, right?

Reply:
Basicly `Q` objects are used when you cannot express DB condition using keyword arguments - https://docs.djangoproject.com/en/3.2/topics/db/queries/#complex-lookups-with-q

Reply:
Are you sure about that? What should be a proper syntax then?

Reply:
`queryset.filter(preferences__accommodation_day_1=False, preferences__accommodation_day_2=False, preferences__accommodation_day_3=False)`
is the same as
`queryset.filter(Q(preferences__accommodation_day_1=False) & Q(preferences__accommodation_day_2=False) & Q(preferences__accommodation_day_3=False))`